### PR TITLE
⚡️ Replaced twitter link by unique link

### PR DESCRIPTION
**Pull request by MrKrisKrisu**
If the ToDo in the Vue templates was also implemented it saves a query to Twitter on every page load.

### CODE REVIEW

**Comment by MrKrisKrisu** [comment]
@todo for @HerrLevin 
Maybe you can do this in this PR before it is merged

**Comment by HerrLevin** [comment]
> @todo for @HerrLevin
> Maybe you can do this in this PR before it is merged

Sorry, but why didn't you just change the `getTwitterUrlAttribute()` to return this link? This would've changed it in vue and in in blade. 😅

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -170,6 +170,9 @@ public static function statusesForUser(User $user): ?LengthAwarePaginator {
                     })->orderByDesc('created_at')->paginate(15);
     }
 
+    /**
+     * @todo remove twitterUrl after implemented ID-Link in vue Template
```
Comment (HerrLevin) [inline app/Http/Controllers/UserController.php:174]: See https://github.com/Traewelling/traewelling/pull/467#issuecomment-886096709

**Review by HerrLevin — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/UserController.php (hunk 1 above)
Comment (MrKrisKrisu) [reply app/Http/Controllers/UserController.php:174]: Don't you think the method would be unnecessary if it just returned a string without doing anything?

Comment (HerrLevin) [reply app/Http/Controllers/UserController.php:174]: Hm. Maybe.

**Review by HerrLevin — APPROVED** [review]

**Pull request merged**